features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/apim_abnormal-backend-time-alert_0.siddhi

```diff
@@ -14,7 +14,7 @@
     limitations under the License.
 */
 
-@App:name("APIM_ABNORMAL_BACKEND_TIME")
+@App:name("apim_abnormal-backend-time-alert_0")
```

Comment:
Can we change the siddhi app names to use underscores as word separators, instead of `-` character? E.g. `apim_abnormal_backend_time_alert_0` instead of `apim_abnormal-backend-time-alert_0`?